web/packages/teleport/src/Users/Users.tsx

```diff
@@ -62,20 +64,30 @@ export function Users(props: State) {
   } = props;
   return (
     <FeatureBox>
-      <FeatureHeader>
+      <FeatureHeader justifyContent="space-between">
         <FeatureHeaderTitle>Users</FeatureHeaderTitle>
         {attempt.isSuccess && (
           <>
             {!InviteCollaborators && (
-              <Button
-                intent="primary"
-                fill="border"
-                ml="auto"
-                width="240px"
-                onClick={onStartCreate}
+              <HoverTooltip
+                tipContent={
+                  canEditUsers
+                    ? ''
+                    : 'You do not have permission to edit users.'
```

Comment:
Should this be

```suggestion
                    : 'You do not have permission to create users.'
```

Reply:
techincally they can create, but the edit is required in the second step. I'm not sure which is better as the "edit" version is more correct, but its on the 'create' button.

Reply:
and i don't want to hide the button, because we don't want to hide features anymore, only display permission errors

Reply:
I think it should make sense from the user's perspective, i.e. I'm trying to create a user and don't have permissions, so I don't have permission to create users. Maybe this could be more vague, "You do not have enough permissions" or something better worded

We should indicate the missing permissions, I do this in TAG:

![image](https://github.com/user-attachments/assets/c4c36e9e-6f5c-4b85-af52-c38ed2885e23)

![image](https://github.com/user-attachments/assets/0d49d10e-f60e-44d5-b234-95b5100c6878)

with a different message if they're able to edit roles themselves (maybe not needed)

![image](https://github.com/user-attachments/assets/a4e1c796-35fb-49df-a7fd-4fd9e0be3fb7)

cc @roraback

Reply:
That popover looks great (and I agree we should show what permissions are missing), but it would be nice if TAG stuck to our design system so that the user's Teleport experience was consistent. We don't to my knowledge have anything in our design system that uses these colors in this way.

Reply:
I'll make a note to update the colours (I think that grey is just a default I haven't changed)